test/compilable/imports/cstuff1.c

```diff
@@ -292,77 +292,89 @@ void test4(int i)
     long long ll = 12L;
     ll = 12UL;
     long double ld = 1.0L;
-    char* s = "hello" "betty";
+    const char* s = "hello" "betty";
 }
 
 /********************************/
 // https://issues.dlang.org/show_bug.cgi?id=21931
-typedef long int T21931a;
-typedef T21931a T21931b;
+typedef long int long_int;
+typedef long_int my_int;
 
 /********************************/
 // https://issues.dlang.org/show_bug.cgi?id=21934
-typedef int T21934 asm("realtype");
-int init21934 asm("realsym") = 1;
-int var21934 asm("realvsym");
-int fun21934() asm("realfun");
+typedef int type asm("realtype");
+int sym asm("realsym") = 1;
+int vsym asm("realvsym");
+int fun() asm("realfun");
 
-void test21934()
+typedef int asmreg;
+void test_asm()
 {
-    typedef int asmreg;
-    register asmreg r1 asm("r1");
-    // asm ignored by C compiler, should be disallowed?
-    asmreg r2 asm("r2");
-
-    register asmreg r3 asm("r3") = 3;
-    // asm ignored by C compiler, should be disallowed?
-    asmreg r4 asm("r4") = 4;
+  register asmreg r1 asm("r1");
+
+  // asm ignored by C compiler, should be disallowed?
+  asmreg r2 asm("r2");
+
+  register asmreg r3 asm("r3") = 3;
+
+  // Uncomment when bug fixed https://issues.dlang.org/show_bug.cgi?id=21948
+  // asm ignored by C compiler, should be disallowed?
+  //asmreg r4 asm("r4") = 4;
 }
 
 /********************************/
 // https://issues.dlang.org/show_bug.cgi?id=21945
 typedef struct {
     long var;
-} S21945;
-S21945 test21945a;
+} TypedefStruct;
+TypedefStruct typedef_var1;
```

Comment:
????

Reply:
Looks like a victim of rebase.

Reply:
Yeah, with tests removed on top as well. I'll refresh and have another look.

Reply:
#12597